docs/conf.py

```diff
@@ -1,32 +1,19 @@
-# -*- coding: utf-8 -*-
+# Configuration file for Sphinx to build our documentation to HTML.
 #
-# Configuration file for the Sphinx documentation builder.
+# Configuration reference: https://www.sphinx-doc.org/en/master/usage/configuration.html
 #
-# This file does only contain a selection of the most common options. For a
-# full list see the documentation:
-# http://www.sphinx-doc.org/en/master/config
 
-# -- Path setup --------------------------------------------------------------
 
-# If extensions (or modules to document with autodoc) are in another directory,
-# add these directories to sys.path here. If the directory is relative to the
-# documentation root, use os.path.abspath to make it absolute, like shown here.
-#
-# import os
-# import sys
-# sys.path.insert(0, os.path.abspath('.'))
+# -- Project specific imports ------------------------------------------------
 
+import datetime
 
-# -- Project information -----------------------------------------------------
 
-project = u'Jupyter ServerProxy'
-copyright = u'2019, Yuvi Panda'
-author = u'Yuvi Panda'
+# -- Project information -----------------------------------------------------
 
-# The short X.Y version
-version = u''
-# The full version, including alpha/beta/rc tags
-release = u'1.0'
+project = "Jupyter Server Proxy"
+copyright = f"{datetime.date.today().year}, Project Jupyter Contributors"
+author = "Project Jupyter Contributors"
```

Comment:
@yuvipanda do you think this the right call of me to make, to update this like that?

You and Ryan are listed as authors of this package via a package.json that is read by setup.py as well still.

```json
  "author": {
    "name": "Ryan Lovett & Yuvi Panda",
    "email": "rylo@berkeley.edu"
  },
```

Reply:
Fine by me here. Should ping ryan too for the other bit :)

Reply:
:+1: I'm not making that change!